roles/ssp/defaults/main.yml

```diff
@@ -371,6 +382,8 @@ ssp_authsources_saml:
     #      class: "saml:PersistentNameID2TargetedID"
     #      attribute: "eduPersonTargetedID"
     #      nameId: no
+    #idp_function: foo()
+    # Optional function which will construct the IdP entity ID
```

Comment:
Remove duplicate definition of `idp_function` param

Reply:
Done